packages/plugin-webpack/plugin.js

```diff
@@ -200,28 +205,26 @@ module.exports = function plugin(config, args) {
       });
 
       if (!args.skipFallbackOutput) {
-        let assetFiles =
-          chain(
-            stats.toJson({
-              assets: false,
-              hash: true,
-            }),
-            ["entrypoints", "main", "assets"]
-          ) || [];
-
-        let jsFile = assetFiles.find((d) => d.endsWith(".js"));
-        let cssFile = assetFiles.find((d) => d.endsWith(".css"));
+        const entrypoints = stats.toJson({assets: false, hash: true}).entrypoints;
 
         //Now that webpack is done, modify the html file to point to the newly compiled resources
-        scriptEl.src = path.posix.join(baseUrl, jsFile);
-        scriptEl.removeAttribute("type");
+        Object.keys(entries).forEach(name => {
 
-        if (cssFile) {
-          let csslink = dom.window.document.createElement("link");
-          csslink.setAttribute("rel", "stylesheet");
-          csslink.href = path.posix.join(baseUrl, cssFile);
-          dom.window.document.querySelector("head").append(csslink);
-        }
+          if (entrypoints[name] !== undefined) {
+            const assetFiles = chain(entrypoints, [name, "assets"]);
+            const script = entries[name].script;
+            const jsFile = assetFiles.find((d) => d.endsWith(".js"));
+            const cssFile = assetFiles.find((d) => d.endsWith(".css"));
+            script.removeAttribute("type");
+            script.src = path.posix.join(baseUrl, jsFile);
```

Comment:
Not related to this PR, just thinking out loud: I think this `path.posix.join` will probably print backslashes on Windows, which is not what we want. We should probably replace this with a web-friendly, forward-slash-only method in the future.

Reply:
`path.posix.join` will force `path` to join with a `/`, both in windows and linux.